runtime/gc_vlhgc/RememberedSetCardBucket.hpp

```diff
@@ -70,6 +62,11 @@ class MM_RememberedSetCardBucket : public MM_BaseNonVirtual {
 public:
 	/* function members */
 private:
+	MMINLINE uintptr_t offsetMask(MM_EnvironmentVLHGC *env)
+	{
+		return (MAX_BUFFER_SIZE * (env->compressObjectReferences() ? sizeof(uint32_t) : sizeof(uintptr_t))) - 1;
```

Comment:
please use MAX_BUFFER_SIZE * MM_RememberedSetCard::cardSize(env->compressObjectReferences()) 

as-is is unclear if it's relying on card size or obj ref size

Reply:
Done.